list.json

```diff
@@ -1,5 +1,51 @@
 {
 	"corpora": {
+		"semeval-2016-2017-task3-subtaskB-eng": {
+			"num_records": -1,
+			"record_format": "dict",
+			"file_size": 6344358,
+			"reader_code": "https://github.com/RaRe-Technologies/gensim-data/releases/download/semeval-2016-2017-task3-subtaskB-eng/__init__.py",
+			"license": "all files released for the task are free for general research use",
+			"description": "",
+			"checksum": "acab4d1c27f9f8f4da3c72825a22ca88",
+			"file_name": "semeval-2016-2017-task3-subtaskB-eng.gz",
+			"read_more": ["http://alt.qcri.org/semeval2017/task3/", "http://alt.qcri.org/semeval2017/task3/data/uploads/semeval2017-task3.pdf", "https://github.com/RaRe-Technologies/gensim-data/issues/18", "https://github.com/Witiko/semeval-2016_2017-task3-subtaskB-english"],
+			"parts": 1
+		},
+		"semeval-2016-2017-task3-subtaskA-unannotated-eng": {
+			"num_records": 189941,
+			"record_format": "dict",
+			"file_size": 234373151,
+			"reader_code": "https://github.com/RaRe-Technologies/gensim-data/releases/download/semeval-2016-2017-task3-subtaskA-unannotated-eng/__init__.py",
+			"license": "These datasets are free for general research use.",
```

Comment:
Do we have a link supporting this conclusion?

Reply:
The datasets were put together from the following files:
1. [QL-unannotated-data-subtaskA.xml.zip](http://alt.qcri.org/semeval2016/task3/data/uploads/QL-unannotated-data-subtaskA.xml.zip)
2. [semeval2016-task3-cqa-ql-traindev-v3.2.zip](http://alt.qcri.org/semeval2016/task3/data/uploads/semeval2016-task3-cqa-ql-traindev-v3.2.zip)
3. [semeval2016_task3_test.zip](http://alt.qcri.org/semeval2016/task3/data/uploads/semeval2016_task3_test.zip)
4. [semeval2017_task3_test.zip](http://alt.qcri.org/semeval2017/task3/data/uploads/semeval2017_task3_test.zip)

Files 2 and 4 contain explicit license notices – see the “License” section of #18. Files 1 and 3 contain no licensing notices, so technically all rights are reserved. However, to me it seems like an oversight on the side of the task authors who did not include instructions to each of the several ZIP archives.